main.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+
+var product = [
+  { image: 'images/photo1.jpg',
+    title: 'Something With Turquoise'
+  },
+  { image: 'images/photo2.jpg',
+    title: 'Risible Rollings'
+  },
+  { image: 'images/photo3.jpg',
+    title: 'Serpentine and Sarcastic'
+  },
+  { image: 'images/photo4.jpg',
+    title: 'Corpulent Claws'
+  },
+  { image: 'images/photo5.jpg',
+    title: 'Scarlet Sartorial Sense'
+  },
+  { image: 'images/photo6.jpg',
+    title: 'Fulsome Fur'
+  },
+  { image: 'images/photo7.jpg',
+    title: 'Zealous Eater'
+  },
+  { image: 'images/photo8.jpg',
+    title: 'Flaming Fuschia'
+  }
+]
+
+var $artworkSection = document.querySelector('.artwork')
+
+document.addEventListener('DOMContentLoaded', renderPhotos(product))
```

Comment:
You are calling renderPhotos here, did you actually want to pass it to the event listener instead?